core/deps/chdr/coretypes.h

```diff
@@ -1,18 +1,34 @@
+#ifndef __CORETYPES_H__
```

Comment:
`#pragma once` is much preferred over `#ifndef guards` and should be used for new code